src/Arcus.Scripting.ApiManagement/Scripts/Import-AzApiManagementApiPolicy.ps1

```diff
@@ -12,5 +12,5 @@ $result = Set-AzApiManagementPolicy -Context $apimContext -ApiId $ApiId -PolicyF
 if ($result) {
     Write-Host "Successfully updated API policy"
 } else {
-    Write-Error "Failed to update API policy, please check parameters"
+    throw "Failed to update API policy, please check parameters"
```

Comment:
Yes and no 😅 . Yes, it's different than what it seems to describe because it 'changes' the error result. No, because it made the unit tests fail locally on my machine because of this and also because the common practice for errors in this library is to `throw` and not (only) write a message to the error stream. (see other scripts). So, it's also a matter of consistency. 

So, in short: with both adding the `Import-Module` statement and slightly alter this error handling with `throw`, we can now run the unit tests successfully locally. Thx for noticing, though 😉

Reply:
It does propose a interesting question on backwards compatibility, though 🤔 . I'm not aware of any current usages of this script and so can't really predict what flow change this would cause...